TUM Campus App/LoginViewController.swift

```diff
@@ -91,6 +92,17 @@ class LoginViewController: UIViewController {
         textField.layer.borderWidth = isHighlighted ? 1 : 0
         textField.layer.borderColor = isHighlighted ? UIColor.red.cgColor : UIColor(hexString: "0xC7C7CD").cgColor
     }
+    
+    private func setupLogo() {
+        let bundle = Bundle.main
+        let nib = bundle.loadNibNamed("TUMLogoView", owner: nil, options: nil)?.flatMap { $0 as? TUMLogoView }
+        guard let view = nib?.first else { return }
+        logoView = view
+        view.frame = CGRect(x: 0, y: 0, width: 100, height: 40)
+        view.widthAnchor.constraint(equalToConstant: 100).isActive = true
+        view.heightAnchor.constraint(equalToConstant: 40).isActive = true
```

Comment:
I'm not sure about how this works on iOS, but on Android we have many different resolutions of different phones. Does this automatically scale down&up? From the removed code below, I would guess that would've been the intention. (Wouldn't it be great to have comments to see that the original author thought? :P )

Reply:
@thellmund any comments on this?

Reply:
@kordianbruck As far as I know, iOS handles these things like so: The Navigation Bar is _always_ 44 pixels high. On high-res devices, it’s simply drawn at 2x (88 device pixels), and on iPhone X at 3x (132 device pixels). Thus, @TG908’s implementation looks good to me.